docs/builds.rst

```diff
@@ -352,32 +349,28 @@ Below you can find the shortened directory structure of our
 (if ``recursive-clone`` was set to ``True``). The clone, compilation and installation directory have
 ``<build_name>@<revision_name>`` as prefix. Additionally, the compilation directory has ``.compile``
 as suffix. The clone directory is located in the ``/develop`` directory, whereas the compilation and
-installation directories are located in the ``/dev-builds`` directory. The internal simexpal files have
-the suffix ``.simexpal``.
+installation directories are located in the ``/dev-builds`` directory. The internal cache that memorizes
+the internal state is the ``.simex.cache`` file, which is found at the level of the ``experiments.yml`` file. 
```

Comment:
I think you can simply remove the last note on where to find the `.simex.cache` file since you've already mentioned that above. Therefore I recommend the change to be reduced to:
```rst
... installation directories are located in the ``/dev-builds`` directory. The internal cache that memorizes
the internal state is the ``.simex.cache`` file.
```